src/ui/ResultLink/ResultLink.ts

```diff
@@ -303,13 +329,19 @@ export class ResultLink extends Component {
     return (this.options.openQuickview || this.isUriThatMustBeOpenedInQuickview()) && QueryUtils.hasHTMLVersion(this.result);
   }
 
-  private parseHrefTemplate(): string {
-    return this.options.hrefTemplate.replace(/\$\{(.*?)\}/g, (value: string) => {
-      let key = value.substring(2, value.length - 1);
+  private parseStringTemplate(template: string): string {
+    if (!template) {
+      return "";
+    }
+    return template.replace(/\$\{(.*?)\}/g, (value: string) => {
+      let key = value.substring(2, value.length-1);
       let newValue = this.readFromObject(this.result, key);
       if (!newValue) {
         newValue = this.readFromObject(window, key);
       }
+      if(!newValue) {
+        this.logger.warn(`${key} is undefined for this result, please verify your syntax.`);
```

Comment:
C'est deux etapes differentes les if
option 1: `this.readFromObject(this.result, key);`
si ca marche pas: `newValue = this.readFromObject(window, key);`
si ca marche toujours pas: `this.logger.warn`